opacity/static-public-share
# Add a 404 page to display when a file has been deleted

### Issue
 - When a publicly shared file is deleted in [Explorer](https://beta-broker.opacitynodes.com:3000/admin/delete), the public URL still shows the file details but not the file itself since it no longer exists.
### Solution
 - When a public URL of a deleted file is visited, a 404 page should be displayed instead of the details of a file that no longer exists.
### How to test locally
 - Clone the repository and and cd into `/front`
 - Install dependencies by running `npm install`
 - Start the project by running `npm start`
 - A deleted file will always have a file size of 0. Therefore, got to `src/shortlink_dev.html` and under the script tag where there is `window.OpacityFile`, change the value of `fileSizeBytes` to '0'.
 - You should now view a 404 page being displayed
### Screenshot
<img width="1435" alt="Screenshot 2022-05-11 at 8 24 02 PM" src="https://user-images.githubusercontent.com/92468573/167909888-9eb44692-54ed-473a-a78c-f2cbcbb6e540.png">
